packages/tunspace/tunspace.defaults

```diff
@@ -8,6 +8,8 @@ config tunspace "tunspace"
   option uplink_netns "uplink"
   option uplink_ifname "br-wan"
   option uplink_mode "bridge"
+  option uplink_ipv4 ""
```

Comment:
Please add a note that its an optional parameter and defaults to DHCP in case no/empty uplink_ipv4 is configured.